cards/src/main/resources/cards/custom/format_spellsource_basic.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Spellsource Basic",
+  "description": "Just Spellsource's own Core set.",
+  "type": "FORMAT",
+  "collectible": true,
+  "sets": [
+    "SPELLSOURCE_BASIC"
```

Comment:
i think we'd need to do a lot more updates to support this format to help me understand